mappings/net/minecraft/entity/ai/brain/ScheduleRule.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS awa net/minecraft/entity/ai/brain/ScheduleRule
+	METHOD a getPriority (I)F
+		ARG 1 time
```

Comment:
Not really too important, but this is specifically time of day, maybe could be named to reflect that.